bin/reply

```diff
@@ -8,7 +8,7 @@ use Reply::App;
 
 =head1 SYNOPSIS
 
-  reply [-lb] [-I dir] [-M mod] [--version] [--help] [--cfg file]
```

Comment:
Another convention would be to have multiple files for the different branches of execution:

    reply --version
    reply --help
    reply [-lb] [-l dir] [-M mod] [--cfg file]

Or to just say "here be options".

    reply --version
    reply --help
    reply [OPTIONS]
    (See "OPTIONS" below.)